src/router.js

```diff
@@ -139,6 +139,88 @@ router.put('/registrations/:id', async (request, response) => {
   }
 });
 
+router.get('/returns', async (request, response) => {
+  try {
+    const returns = await Return.findAll();
+    if (returns) {
+      // If they are, send back the finalised registrations.
+      response.status(200).send(returns);
+    } else {
+      response.status(404).send({message: `No returns found.`});
+      return;
+    }
+  } catch (error) {
+    response.status(500).send({error});
+  }
+});
+
+router.get('/registrations/:id/return', async (request, response) => {
```

Comment:
I think we should be consistent with the singulars & plurals.
`registrations` vs `return`

Have a look at our MIDAS APIs and see what approach we took there.